src/main/java/com/exactpro/th2/common/schema/factory/CommonFactory.java

```diff
@@ -235,7 +242,13 @@ public static CommonFactory createFromArguments(String... args) {
 
             String configs = cmd.getOptionValue("configs");
 
-            if(cmd.hasOption("namespace") && cmd.hasOption("boxName")) {
+            if ((cmd.hasOption("namespace") && cmd.hasOption("boxName")) && cmd.hasOption("contextName")){
+                String namespace = cmd.getOptionValue("namespace");
+                String clusterName = cmd.getOptionValue("clusterName");
```

Comment:
```suggestion
                String clusterName = cmd.getOptionValue("contextName");
```
Please register contextName in options and create a variable for every instance of Option class to take argument name from them. This approach provides to avoid incorrect name problems.